src/status_im/contexts/syncing/syncing_devices_list/view.cljs

```diff
@@ -9,6 +9,14 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn go-back
+  []
+  (rf/dispatch [:navigate-back]))
+
+(defn open-setup-syncing
+  []
+  (rf/dispatch [:open-modal :settings-setup-syncing]))
```

Comment:
Why do we need to create a function for something that's called once?
We can have the dispatch on the handler in an anonymous function. Fewer lines is always better

Reply:
Even though it's only called once, there's value in defining these handlers outside the react context as they don't depend on any state/props and we don't ever have to re-create them. The alternative could be to define them with use-callback inside the component, but there's a drawback that they have to be memoized while here they don't.